### ISSUE TITLE
Install issue "connecting to internet" 

### ISSUE THREAD

**Issue by good4y0u** [brave/brave-browser#6025]
## Description 

Installing leads to an endless "connecting to internet" then an error. Tested on two different machines on two different networks. 

## Steps to Reproduce

1- bravebrowserinstaller ; download and install from https://laptop-updates.brave.com/latest/winx64

-first attempt not working I disabled all firewalls ; still didnt work

-attempted install "as admin" didn't work either

2- "on your marks" ->"connecting to internet" ->wait a long time -> error 0xa0430721

- when looking this up it appears to be a 'chrome' error not a brave specific one.

I'm not sure, but given the application is making it out of my network, could this be a brave issue?

anyone have a fix for this, unfortunately with no windows based standalone installer I can't just use that.

## Actual result:

error out of download for install

## Expected result:
installation and download success 

## Reproduces how often: 

easily reproduced

**Comment by rebron** [comment]
Hi @good4y0u Can you try the standalone installer -> https://github.com/brave/brave-browser/releases/tag/v0.68.141
You want this one -> BraveBrowserStandaloneSetup.exe

**Comment by good4y0u** [comment]
> Hi @good4y0u Can you try the standalone installer -> https://github.com/brave/brave-browser/releases/tag/v0.68.141
> You want this one -> BraveBrowserStandaloneSetup.exe

So at the time this was giving me a "your browser is managed" tag. Which was odd, because my chrome installation did not have that tag, nor did my previous brave version. I ended up waiting for a day and running the brave install again and it worked. According to some reddit users the problem was reproducible for them too. I just picked a bad day to do a re-install.
